test/ws-prefix-rewrite-core.js

```diff
@@ -32,10 +32,11 @@ async function proxyServer (t, backendURL, backendPath, proxyOptions, wrapperOpt
   return [frontend, await frontend.listen(0)]
 }
 
-async function processRequest (t, frontendURL, path, expected) {
+async function processRequest(t, frontendURL, path, expected) {
   const url = new URL(path, frontendURL)
   t.comment('ws connecting to ' + url.toString())
-  const ws = new WebSocket(url)
+  const wsUrl = url.href.replace('http://', 'ws://')
```

Comment:
why do it in so many [different](https://github.com/fastify/fastify-http-proxy/pull/172/files#diff-269802b67ef10bddd18881f6f23f2f9555974a2ccfed0ead1ef4eb79ce889af7R37) [ways](https://github.com/fastify/fastify-http-proxy/pull/172/files#diff-fda8f0e13abe3d5c5685ad895fa528dd6067c41b66b6ebc6d1de71ca9a704d8dR3)? let's pick one and use that consistently throughout

Reply:
👍